Graphics/Sudbury/WirePackages.hs

```diff
@@ -26,37 +24,33 @@ data WirePackage = WirePackage
   , wirePackagePayload :: B.ByteString
   } deriving (Eq, Show)
 
--- | Construct a wayland wire wirePackage
-wirePack :: WirePackage -> BB.Builder
-wirePack msg = wirePackBuilder
-  (wirePackageSender msg)
-  (wirePackageSize msg)
-  (wirePackageOpcode msg)
-  (BB.byteString $ wirePackagePayload msg)
-
--- | Construct a wayland wire wirePackage with payload given as a 'Builder'.
-wirePackBuilder :: Word32 -> Word16 -> Word16 -> BB.Builder -> BB.Builder
-wirePackBuilder sender size opcode payload =
-  BBE.word32Host sender
-  <>
-  -- FIXME make byte order portable here
```

Comment:
I forgot what this FIXME was about, it seems to be doing the right thing. So yes this comment can probably be dropped.